contracts/test.sol

```diff
@@ -74,28 +74,19 @@ contract TestUtils is DSTest {
         fail();
     }
 
-    // Constrict values to a range, inclusive of min and max values. Source: solmate
+    // Constrict values to a range, inclusive of min and max values.
     function constrictToRange(
         uint256 x,
         uint256 min,
         uint256 max
     ) internal pure returns (uint256 result) {
         require(max >= min, "MAX_LESS_THAN_MIN");
 
-        uint256 size = max - min;
+        if (min == max) return min;  // A range of 0 is effectively a single value.
 
-        if (size == 0) return min;            // Using max would be equivalent as well.
-        if (max != type(uint256).max) size++; // Make the max inclusive.
+        if (min == 0 && max == type(uint256).max) return x;  // The entire uint256 space is effectively x.
 
-        // Ensure max is inclusive in cases where x != 0 and max is at uint max.
-        if (max == type(uint256).max && x != 0) x--; // Accounted for later.
-
-        if (x < min) x += size * (((min - x) / size) + 1);
-
-        result = min + ((x - min) % size);
-
-        // Account for decrementing x to make max inclusive.
-        if (max == type(uint256).max && x != 0) result++;
+        return (x % ((max - min) + 1)) + min;  // Given the above exit conditions, `(max - min) + 1 <= type(uint256).max`.
```

Comment:
This would fail if `min` is non-zero while `x` and `max` are both the maximum `uint256`.

Scenario: `min = 1, max = x = type(uint256).max`
Expected: `result = max` (since max is inclusive)
Result: `result = (max % ((max - 1) + 1)) + 1 = (max % max) + 1 = 0 + 1 = 1`

Reply:
That's the expected outcome though. since the range/size it `type(uint256).max - 1`, x would loop over to `1`.

Reply:
If the value `x` is already within the range (`x >= min && x <= max`), it shouldn't be affected, right?